tests/integration/usage_statistics/test_usage_statistics_messages.py

```diff
@@ -0,0 +1,246 @@
+        }
+    ],
+    "data_context.open_data_docs": [
+        {
+            "event_payload": {},
+            "event": "data_context.open_data_docs",
+            "success": True,
+            "version": "1.0.0",
+            "event_time": "2020-06-25T16:08:28.070Z",
+            "data_context_id": "00000000-0000-0000-0000-000000000002",
+            "data_context_instance_id": "10000000-0000-0000-0000-000000000002",
+            "ge_version": "0.11.5.manual_testing",
+            "x-forwarded-for": "00.000.00.000, 00.000.000.000",
+        }
+    ],
+    "data_context.build_data_docs": [
+        {
+            "event_payload": {},
+            "event": "data_context.build_data_docs",
+            "success": True,
+            "version": "1.0.0",
+            "event_time": "2020-06-25T16:08:24.349Z",
+            "data_context_id": "00000000-0000-0000-0000-000000000002",
+            "data_context_instance_id": "10000000-0000-0000-0000-000000000002",
+            "ge_version": "0.11.5.manual_testing",
+            "x-forwarded-for": "00.000.00.000, 00.000.000.000",
+        }
+    ],
+    "data_context.save.expectation.suite": [
+        {
+            "event_payload": {
+                "anonymized_expectation_suite_name": "4b6bf73298fcc2db6da929a8f18173f7"
+            },
+            "event": "data_context.save_expectation_suite",
+            "success": True,
+            "version": "1.0.0",
+            "event_time": "2020-06-25T16:08:23.570Z",
+            "data_context_id": "00000000-0000-0000-0000-000000000002",
+            "data_context_instance_id": "10000000-0000-0000-0000-000000000002",
+            "ge_version": "0.11.5.manual_testing",
+            "x-forwarded-for": "00.000.00.000, 00.000.000.000",
+        }
+    ],
+    "data_context.add_datasource": [
+        {
+            "event_payload": {
+                "anonymized_name": "c9633f65c36d1ba9fbaa9009c1404cfa",
+                "parent_class": "PandasDatasource",
+            },
+            "event": "data_context.add_datasource",
+            "success": True,
+            "version": "1.0.0",
+            "event_time": "2020-06-25T16:08:16.030Z",
+            "data_context_id": "00000000-0000-0000-0000-000000000002",
+            "data_context_instance_id": "10000000-0000-0000-0000-000000000002",
+            "ge_version": "0.11.5.manual_testing",
+            "x-forwarded-for": "00.000.00.000, 00.000.000.000",
+        }
+    ],
+    # TWEAK TO BE LIKE ABOVE
+    # "cli.store.list",
+    # "cli.project.check_config",
+    # "cli.validation_operator.run",
+    # "cli.validation_operator.list",
+    # "cli.tap.new",
+    # "cli.docs.list",
+    # "cli.docs.build", --> has this been changed to data_context.build_data_docs? #TODO check if this is the case
+    # "cli.datasource.profile",
+    # "cli.datasource.list",
+    # "cli.datasource.new" --> has this been changed to cli.new_ds_choice? #TODO check if this is the case
```

Comment:
This was just some confusion on my part (Both these #TODOs are actually separate and real `send_usage_message()` calls and should go away). But it did make me think of how careful we would need to be in naming the events.